ui/src/Dashboard/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react'
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+// import loginSer from "./login.service";
```

Comment:
No commented code please. And there is no service concept in react.

Reply:
removed imports